src/components/Post/Details.tsx

```diff
@@ -19,6 +21,18 @@ export const postDetailsTestIds = {
 };
 
 const PostDetails = ({ post }: PostDetailsProps) => {
+  const renderCommaSeparatedTags = () => {
+    const tags = post.tags.map((tag) => (
+      <Link key={tag} href={`/tags/${tag}`}>
+        <a className="hover:underline" href={`/tags/${tag}`}>
+          {tag}
+        </a>
+      </Link>
+    ));
+
+    return intersperse(tags, ', ');
```

Comment:
@tyrro Instead of a helper here, should we use CSS to solve this?
References: [last (:last-child)](https://tailwindcss.com/docs/hover-focus-and-other-states#last) and [pseudo-element](https://tailwindcss.com/docs/content)
Demo: https://jsfiddle.net/wvkpomxs/4/

It looks a little bit weird to me in tailwind ☠️
If it's just only CSS it's more tidy

Reply:
Thanks for the wonderful snippet, but as we have got rid of this `intersperse` function in favor of daisyUI badges in 7474e9a4d9524a404e431800cc6495c8d93fc6f5, we no longer need it!

Reply:
Cool 👍